src/System.Diagnostics.Process/tests/ProcessStreamReadTests.cs

```diff
@@ -237,8 +237,8 @@ public void TestClosingStreamsAsyncDoesNotThrow()
 
             // On netfx, the handler is called once with the EventArgs as null, even if the process writes nothing to the pipe.
             // That's fine, so we'll ignore it here.
-            p.OutputDataReceived += (s, e) => { Assert.True(PlatformDetection.IsFullFramework && e == null, "OutputDataReceived called after closing the process"); };
-            p.ErrorDataReceived += (s, e) => { Assert.True(PlatformDetection.IsFullFramework && e == null, "ErrorDataReceived called after closing the process"); };
+            p.OutputDataReceived += (s, e) => { Assert.True(PlatformDetection.IsFullFramework && string.IsNullOrWhiteSpace(e?.Data), "OutputDataReceived called after closing the process"); };
+            p.ErrorDataReceived += (s, e) => { Assert.True(PlatformDetection.IsFullFramework && string.IsNullOrWhiteSpace(e?.Data), "ErrorDataReceived called after closing the process"); };
```

Comment:
What's the reason to ignore also white space lines? Does it actually happen on any platform? If not, I would vote for original (more strict) condition.

Interestingly, the `null` upon closing stream is [documented](https://msdn.microsoft.com/en-us/library/system.diagnostics.datareceivedeventhandler(v=vs.110).aspx). I wonder why .NET Core is different - @wtgodbe @joperezr. any idea?

Reply:
I misread my own logging in this case. I had to change the code a bit since Assert.True surfaces as a serialization error due to xUnit not marking their exceptions as serializable [which is known and won't change](https://github.com/xunit/xunit/issues/298), and then misread what I had logged. On my machine `e == null` is true for Output but false for Error and `e.Data == null` is true for Error. My misreading made me think `e.Data` was an empty string.